src/UCTSearch.cpp

```diff
@@ -43,12 +45,12 @@ constexpr int UCTSearch::UNLIMITED_PLAYOUTS;
 
 class OutputAnalysisData {
 public:
-    OutputAnalysisData(const std::string& move, int visits, int winrate, std::string pv) :
-        m_move(move), m_visits(visits), m_winrate(winrate), m_pv(pv) {};
+    OutputAnalysisData(const std::string& move, int visits, int winrate, std::string N_num, std::string pv) :
+        m_move(move), m_visits(visits), m_winrate(winrate), m_N_num(N_num), m_pv(pv) {};
```

Comment:
What's m_N_num?

Reply:
It's the policy prior; mylizzie needs it to display the policy prior together with the move, principal variation, winrate and visit count in a table. Originally, the N_num output breaks Lizzie, but after https://github.com/featurecat/lizzie/pull/329 it's now compatible with Lizzie. That PR also added support to display stm_komi and opp_komi in Lizzie.

Reply:
Aha! We should split off this feature. (And rename it to something clearer)

Do you want to do this?

It's generally useful outside this pull, and it will make this diff smaller again.